cmd/kubelet/app/options/options.go

```diff
@@ -544,6 +539,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.StringVar(&c.ContainerLogMaxSize, "container-log-max-size", c.ContainerLogMaxSize, "<Warning: Alpha feature> Set the maximum size (e.g. 10Mi) of container log file before it is rotated. This flag can only be used with --container-runtime=remote.")
 	fs.Int32Var(&c.ContainerLogMaxFiles, "container-log-max-files", c.ContainerLogMaxFiles, "<Warning: Alpha feature> Set the maximum number of container log files that can be present for a container. The number must be >= 2. This flag can only be used with --container-runtime=remote.")
 
+	fs.StringSliceVar(&c.AllowedUnsafeSysctls, "allowed-unsafe-sysctls", c.AllowedUnsafeSysctls, "Comma-separated whitelist of unsafe sysctls or unsafe sysctl patterns (ending in *). Use these at your own risk. Sysctls feature gate is enabled by default.")
```

Comment:
What is the status of this feature? (alpha, beta, GA?)
Which feature gate applies to it?

"Use these at your own risk" makes me a little nervous moving the flag into a beta API... but that's potentially just an old comment that is no longer relevant.

Reply:
Going to Beta (PR almost merged). And yes, the `"Use these at your own risk"` can be removed. I will update the PR once rebased.

Reply:
The `"Use these at your own risk" ` is actually intentional. By default, no unsafe sysctl is allowed. Though, there are some applications that require (or work better) with some unsafe sysctls allowed. So it's up to an admin to (re)consider which unsafe sysctls are "ok" at his own risk.